README.md

```diff
@@ -0,0 +1,60 @@
+# Shavar list testing using test pages
+Instructions on how to create test profiles: https://wiki.mozilla.org/Services/TrackingProtection/Shavar_Server_-_Testing#SET_CUSTOM_PREFS
+
+### https://mozilla.github.io/tracking-test/ads.html
+1. Create a BASE profile
+2. Restart browser
+3. Navigate to https://mozilla.github.io/tracking-test/ads.html
+
+expected result: fox displays, confirming page is serving correct content
```

Comment:
should we clarify specifically what we mean by "correct content?"

Reply:
The correct content is that the fox displays. The test pages serve limited functionality- they show if the content is correct or not. I don't think this would be the place to explain why or how the test pages are created, only that they are working based on the expected results.